service_container.module

```diff
@@ -8,28 +8,36 @@
 // Core Hooks
 
 /**
+ * Implements hook_stream_wrappers_alter().
+ */
+function service_container_stream_wrappers_alter(&$wrappers) {
+  if (class_exists('ServiceContainer')) {
+    ServiceContainer::init();
+  }
+}
+
+/**
+ * Implements hook_enable().
+ */
+function service_container_enable() {
+  if (class_exists('ServiceContainer')) {
+    ServiceContainer::reset();
```

Comment:
That however means:

a) This runs only when service_container is enabled.
b) We don't have _any_ service container afterwards anymore.

Should we not re-init after module enabling?

Hence:

```
function service_container_modules_enabled() {
  if (class_exists('ServiceContainer')) {
    ServiceContainer::reset();
    ServiceContainer::init();
  }
}
```